src/components/TopBar/TopBar.tsx

```diff
@@ -663,7 +663,11 @@ const TopBar = ({
                     appLicense={appLicense}
                     globalVersionPromise={globalVersionPromise}
                     additionalModulesPromise={additionalModulesPromise}
-                    logo={logoAboutDialog}
+                    logo={
```

Comment:
Keep the behavior that you implemented. Fix only the app name in the details section.